bin/populate_sample_db

```diff
@@ -69,8 +69,11 @@ for num in range(1, num_teams + 1):
         for property_num in range(1, num_properties + 1):
             print("\t\t\t\tCreating Property " + str(property_num))
             property_obj = Property(name="prop" + str(property_num), value="randval" + str(property_num), service=service)
+            property_obj = Property(name="IP Address", value=ip, service=service)
```

Comment:
I think we can delete this line since we're creating an IP Address property already